ietf/doc/templatetags/document_type_badge.py

```diff
@@ -0,0 +1,23 @@
+# Copyright The IETF Trust 2015-2020, All Rights Reserved
+from django import template
+from django.template.loader import render_to_string
+# from django.urls import reverse
+
+# from ietf.group.models import Group
+# from ietf.name.models import GroupTypeName
+
+register = template.Library()
+
+
+@register.simple_tag
+def document_type_badge(doc, snapshot):
+    if doc.type == "rfc":
+        return render_to_string(
+            "doc/badge/doc-badge-rfc.html",
+            {"doc": doc, "snapshot": snapshot},
+        )
+
+    return render_to_string(
+        "doc/badge/doc-badge-other.html",
```

Comment:
this should be called doc-batch-draft - it's only badging Internet-drafts.

It might be worth using this kind of shape:

```
if doc.type == "rfc":
   ...
elif doc.type == "draft":
  ...
else:
   return no badge, raise an error in dev/test, and log a warning in production.
```